keps/prod-readiness/sig-scheduling/3094.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3094
+alpha:
+  approver: ""
```

Comment:
I can take this once you have SIG-level approval

Reply:
thanks